docs/resources/item_attachment.md

```diff
@@ -0,0 +1,43 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
```

Comment:
I assume this file was not generated. You will need to content of [this folder](https://github.com/maxlaverse/terraform-provider-bitwarden/tree/1530e912ead5200a7091fd87b444b139e36b412c/examples/resources/bitwarden_item_attachment) in order to have the import section working, and then run `make docs`